django/forms/forms.py

```diff
@@ -482,6 +482,17 @@ def visible_fields(self):
         """
         return [field for field in self if not field.is_hidden]
 
+    def get_initial_for_field(self, field, field_name):
+        """
+        Return initial data for field on form. Return the form's initial data
```

Comment:
For the second sentence, maybe: "Use initial data from the form or the field, in that order."